src/In.ProjectEKA.HipService/Startup.cs

```diff
@@ -81,7 +81,7 @@ public void ConfigureServices(IServiceCollection services)
                 .AddHangfire(config => { config.UseMemoryStorage(); })
                 .AddSingleton<IEncryptor, Encryptor>()
                 .AddSingleton<IPatientRepository>(new PatientRepository("demoPatients.json"))
-                .AddSingleton<ICollect>(new Collect("demoPatientCareContextDataMap.json"))
+                .AddSingleton<ICollect>(new Collect("http://localhost:8003/getData", HttpClient))
```

Comment:
Can we fetch this URL from appsettings.json?

Reply:
I think we were sharing the sample. If you want I can put it in.

Reply:
yes, please put it in appsettings.json

Reply:
Done